docs/website/docs/walkthroughs/run-in-snowflake/run-in-snowflake.md

```diff
@@ -0,0 +1,6 @@
+---
+title: Run dlt in Snowflake
+description: Run dlt in Snowflake Native App
+keywords: [snowflake, native app, dlt]
+---
+
```

Comment:
Let's not leave it completely empty! Could you just add a paragraph that there are several different ways to run dlt directly on the Snowflake.